environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -144,11 +144,29 @@ private ValidationResult validateEntitlement(ValidationResultBuilder validationR
                         build();
             }
         }
+        String encryptionKeyResourceGroupName = azureResourceEncryptionParametersDto.getEncryptionKeyResourceGroupName();
+        if (Objects.nonNull(encryptionKeyResourceGroupName)) {
+            if (!entitlementService.isAzureDiskSSEWithCMKEnabled(accountId)) {
+                LOGGER.info("Invalid request, CDP_CB_AZURE_DISK_SSE_WITH_CMK entitlement turned off for account {}", accountId);
```

Comment:
can we somehow phrase it differently? E.g CDP_CB_AZURE_DISK_SSE_WITH_CMK is not enabled on your account. Please contact with Cloudera support if you would like to use Azure encryption on CDP platform ... or something like this